analyses/cell-types-annotation/README.md

```diff
@@ -0,0 +1,65 @@
+# Pipeline for annotating cell types for sc-/sn-RNA-Seq Analysis in 10X Genomics data
+
+## Usage
+
+To run the script in this module from the command line sequentially, use:
+
+```
+bash run-cell-types-annotation.sh
+```
+
+`run-cell-types-annotation.sh` is designed to be run as if it was called from this module directory even when called from outside of this directory.
+
+Parameters according to the project and analysis strategy will need to be specified in the following scripts:
+- `project_parameters.Config.yaml` located at the `root_dir`
+- `run-cell-types-annotation.R`: define `root_dir`, `data_dir`, and define params for: `01-cell-types-annotation-SingleR-broad.Rmd` and `02-cell-types-annotation-SingleR-fine.Rmd` in the `rmarkdown::render code chunk`
```

Comment:
I'm not sure I follow what is being specified in the `run-cell-types-annotation.R` script. It seems like all the params being passed to render `01-cell-types-annotation-SingleR-broad.Rmd` and `02-cell-types-annotation-SingleR-fine.Rmd` are being defined in the yaml. Maybe this could be edited to make it a bit more clear?

Reply:
Fixed here a77e41362c6f9a39f3cd25be045271052fdb72d2.